deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -109,7 +109,7 @@ def delete_screenshots(localizations, screenshots_per_language, tries: 5)
       end
     end
 
-    def upload_screenshots(localizations, screenshots_per_language, tries: 5)
+    def upload_screenshots(localizations, screenshots_per_language, tries: 5, timeout_seconds: 3600)
```

Comment:
Any specific reason you added a default value to this new param? Was it to maintain backwards compatibility in case anyone is using this method?

You covered all the call sites that are invoking this method, passing the timeout from the user's options, so I think we could remove this default value in favor of the one declared in the options "default value" — this makes the default value have a single source of truth, which I think it's better. This method shouldn't be used by users anyway probably, they should be calling `deliver` directly instead, so I think this is safe, also.

What do you think? 🙏

Reply:
Totally agree, it should make code better 👍